active/radiobuttons/RadioButtons_Spec.md

```diff
@@ -0,0 +1,271 @@
+# Background
+Radio buttons are used for mutually exclusive options in an app. When the users selects
+one of a group, the previously-checked button is unchecked. Consequently, radio buttons
+must be grouped, which is done with the Xaml [RadioButton](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.RadioButton)
+control by setting the GroupName property on each button to the same value.
+
+That mechanism is awkward, and doesn't by default give the right keyboarding or
+accessibility behavior.
+
+The RadioButtons control in this spec is introduced to help solve those issues.
+It's like a ListView control, but the items in the list are RadioButton controls.
+
+# Description
+
+RadioButtons is a new control that enables you to create groups of RadioButton elements easily,
+while also correctly supporting keyboarding and narrator functionality.
+
+# Examples
+
+The following example demonstrates how to create a small collective group of RadioButton elements
+using the RadioButtons control.
+
+```xaml
+<muxc:RadioButtons Header="App Mode" SelectedIndex="2">
+    <x:String>Item 1</x:String>
+    <x:String>Item 2</x:String>
+    <x:String>Item 3</x:String>         
+</muxc:RadioButtons>
+```
+
+![alt text](DefaultRadioButtonGroup.png)
+
+## DataBinding with RadioButtons
+An example of simple data binding of the ItemsSource with RadioButtons.
+
+```xaml
+<muxc:RadioButtons Header="App Mode" ItemsSource="{x:Bind radioButtonItems}" />
+```
+
+```C#
+public sealed partial class MainPage : Page
+{
+    public class OptionDataModel
+    {
+        public string Label;
+        public override string ToString()
+        {
+            return Label;
+        }
+    }
+
+    List<OptionDataModel> radioButtonItems;
+
+    public MainPage()
+    {
+        this.InitializeComponent();
+
+        radioButtonItems = new List<OptionDataModel>();
+        radioButtonItems.Add(new OptionDataModel() { Label = "Item 1" });
+        radioButtonItems.Add(new OptionDataModel() { Label = "Item 2" });
+        radioButtonItems.Add(new OptionDataModel() { Label = "Item 3" });
+    }
+}
+```
+
+## Vertical and horizontal RadioButton lists
+In instances where there is more that one column of RadioButton items within a single RadioButtons list control, the list will flow a column-major layout.
```

Comment:
```suggestion
In instances where there is more that one column of RadioButton items within a single RadioButtons list control, the list will flow in a column-major layout.
```

Maybe I'm the only one confused by this column-major wording... I find the "fill itself vertically" statement easier to understand.

Reply:
We define and use column-major in the docs [here](https://docs.microsoft.com/en-us/windows/uwp/design/input/keyboard-interactions#wrapping-homogeneous-list-and-grid-view-items) - so I'm attempting to use the same terminology in this doc as well.